src/controllers/management-controller.ts

```diff
@@ -452,11 +426,22 @@ export class ManagementController {
           "@context": this.defaultContextValues,
         },
       })
-      .then((body) => jsonld.expand(body))
-      .then((expanded) =>
-        (expanded as Array<any>).map((it) =>
-          Object.assign(new TransferProcess(), it),
-        ),
-      );
+      .then(body => this.expandArray(body, () => new TransferProcess()));
+  }
+
+  private async expand<T extends JsonLdObject>(body: any, newInstance: (() => T)): Promise<T> {
+    const expanded = await jsonld.expand(body);
+    var instance = Object.assign(newInstance(), expanded[0]);
+    instance._compacted = body;
+    return instance;
+  }
+
+  private async expandArray<T extends JsonLdObject>(body: any, newInstance: (() => T)): Promise<T[]> {
+    const expanded = await jsonld.expand(body);
+    return (expanded as Array<any>).map((element, index) => {
+      var instance = Object.assign(newInstance(), element);
+      instance._compacted = body[index];
+      return instance;
+    });
```

Comment:
I suggest that we move this block of code to a different utilities file or class to keep this file only for containing management controller methods